src/endpoint.cpp

```diff
@@ -44,9 +44,22 @@ namespace oxen::quic
     {
         for (const auto& c : conns)
             c.second->io_trigger->on<uvw::AsyncEvent>(async_cb);
+    }
 
-        // for (const auto& c : conns)
-        //     c.second->io_ready();
+    void Endpoint::get_active()
```

Comment:
should probably be something like `get_active_conns`...well, if in the future it returns them; I guess `print_active_conns` would be ok for now.

Reply:
Yeah returning them was the purpose of the unfortunately named “get_conns” that you commented on below (I agree we should change the name to something more descriptive). 

This function is intended to be user facing, since servers explicitly calling open_stream can specify the connection. The stream can also be extracted as soon as it’s ready via callback like in 006 and 007. 

I also agree we should change the name of this, will change that